### PR TITLE
Show more comment info in admin page

### PR DESCRIPTION
**Pull request by int-y1**
add column for "votes" and "hidden"

Capture

### CODE REVIEW

File: judge/admin/comments.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class CommentAdmin(VersionAdmin):
         (None, {'fields': ('author', 'page', 'parent', 'time', 'score', 'hidden')}),
         (_('Content'), {'fields': ('body',)}),
     )
-    list_display = ['author', 'linked_page', 'time']
+    list_display = ['author', 'linked_page', 'time', 'score', 'hidden']
```
Comment (kiritofeng) [inline judge/admin/comments.py:25]: Is it possible to rename "Hide the comment" to "Hidden"?

Comment (int-y1) [reply judge/admin/comments.py:25]: done, see the change in `Comment.hidden.verbose_name`

**Review by kiritofeng — APPROVED** [review]
LGTM, thanks!

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**